soccer_vision_2d_msgs/README.md

```diff
@@ -2,26 +2,6 @@
 
 This package provides 2d messages for visually detected objects on a soccer field, such as balls, field markings and robots.
 
-## Messages (.msg)
+## Documentation
 
-* Ball
-* BallArray
-* FieldBoundary
-* Goalpost
-* GoalpostArray
-* MarkingEllipse
-* MarkingIntersection
-* MarkingSegment
-* MarkingArray
-* Obstacle
-* ObstacleArray
-* Robot
-* RobotArray 
-
-## Examples
-
-![](images/soccer_vision_2d_msgs_visualized.png)
-
-![](images/obstacle_fieldboundary_markings_visualized.png)
-
-![](images/penalty_spots_and_more.png)
\ No newline at end of file
+For documentation, see [Soccer Vision 2D Msgs](https://soccer-interfaces.readthedocs.io/en/latest/soccer_vision_2d_msgs.html).
```

Comment:
Is it correct, that this link does not work (yet)?

Reply:
That link should be working already.